src/Robo/Commands/Setup/SettingsCommand.php

```diff
@@ -286,4 +286,29 @@ public function hashSalt() {
     }
   }
 
+  /**
+   * Writes a deployment_identifier to ${repo.root}/deployment_identifier.
+   *
+   * @command drupal:deployment-identifier:init
+   * @aliases ddii
+   *
+   * @throws \Acquia\Blt\Robo\Exceptions\BltException
+   */
+  public function createDeployId($id = '') {
+    if (!$id) {
```

Comment:
Arguments should be required. If you want an option, use an $options array. See https://github.com/consolidation/annotated-command#option-default-values

Reply:
Ack